weblate/addons/models.py

```diff
@@ -372,3 +393,19 @@ def store_post_load_handler(sender, translation, store, **kwargs) -> None:
         (translation, store),
         translation=translation,
     )
+
+
+class AddonActivityLog(models.Model):
+    addon = models.ForeignKey(Addon, on_delete=models.deletion.CASCADE)
+    component = models.ForeignKey(Component, on_delete=models.deletion.CASCADE)
+    event = models.IntegerField(choices=AddonEvent.choices)
+    created = models.DateTimeField(auto_now_add=True)
+    details = models.JSONField(default=dict)
```

Comment:
Maybe add severity or at least indication whether this is an error? Currently, there is no way to distinguish errors besides looking at the `details`.